registry/domain/services/registry_blockchain_util.py

```diff
@@ -1,14 +1,12 @@
-from unittest.mock import Mock
-
 import web3
 
 from common.blockchain_util import BlockChainUtil
+from common.boto_utils import BotoUtils
 from common.exceptions import MethodNotImplemented
+from common.logger import get_logger
 from common.utils import ipfsuri_to_bytesuri
-from registry.config import NETWORK_ID, NETWORKS, BLOCKCHAIN_TEST_ENV
+from registry.config import NETWORK_ID, NETWORKS, BLOCKCHAIN_TEST_ENV, REGION_NAME
```

Comment:
I dont see region_name in the config file

Reply:
[registry/config.py](https://github.com/singnet/snet-marketplace-service/pull/1341/files/e436ae6b92be6114a2a1c9f6035b3abd6d920633#diff-3fd7e09cee517d56d9d845e8c272c1cc) it is there in line no 33